packages/app/config/environment.js

```diff
@@ -63,8 +63,11 @@ module.exports = function(environment) {
 
   if (environment === 'production') {
     ENV['ember-cli-mirage'] = {
-      enabled: false //set to true for gh-pages
+      enabled: process.env.BUILD_NAVI_DEMO === 'true'
     };
+    if (process.env.BUILD_NAVI_DEMO === 'true') {
```

Comment:
Had to add this because the demo app was producing the wrong asset URL for github pages